test/specs/wrapper/find.spec.js

```diff
@@ -441,7 +468,7 @@ describeWithShallowAndMount('find', mountingMethod => {
     const wrapper = mountingMethod(compiled)
     const a = wrapper.find('a')
     const message =
-      '[vue-test-utils]: $ref selectors can only be used on Vue component wrappers'
+      '[vue-test-utils]: $ref selectors can used on Vue component wrappers'
```

Comment:
Is this an expected modification?

Reply:
I dont remember making this one, but it does improve the error message 🤔

Reply:
Should probably read "can **be** used on" (missing "be"), though.

Reply:
done, updated. (comment still show old version, but if you view "files changed" is it new updated one)